app/controllers/admin/users_controller.rb

```diff
@@ -15,8 +15,9 @@ def new
 
     # GET /admin/users
     def index
+      default_params = { 'users.created_at' => 'desc' }
```

Comment:
Perhaps move to a private method `default_order_params`?

Reply:
Great suggestion. Done.